usagov/vote-gov
# Vote-254: Update to Yupik string for "Other ways to Vote"

## Jira ticket

[Vote-254](https://cm-jira.usa.gov/browse/VOTE-254)

## Description

This Pr will update the "Other ways to vote" string within the Yupik translation on state pages

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. Vist [Preview Link](https://cg-9e8debaf-b030-4825-a43c-cb2bc850c96c.sites.pages.cloud.gov/preview/usagov/vote-gov/bug/vote-254-yupik-update/ypk/register/ak/) and the string above the drop-down for NVRF download should match `Ilaneng ilatelleghqaneng`

## Checklist for the Developer




## Checklist for the Peer Reviewers
